source/configuration/modules/omfile.rst

```diff
@@ -1,434 +1,441 @@
+
+The extra headers considerably degrade compression, files with this option set to "on" may be four to five times as large as files processed in "off" mode.
+
+==============  ====================
+flushInterval:  "seconds"
+Default:        "1"
+Legacy:         $OMFileFlushInterval
+==============  ====================
+
+Defines, in seconds, the interval after which unwritten data is flushed.
+
+=============  ===================
+asyncWriting:  "on|off"
+Default:       "off"
+Legacy:        $OMFileASyncWriting
+=============  ===================
+
+if turned *on*, the files will be written in asynchronous mode via a
+separate thread. In that case, double buffers will be used so that
+one buffer can be filled while the other buffer is being written.
+   
+Note that in order to enable *flushInterval*, this parameter must be *on*. Otherwise, the flush interval will be ignored.
+Also note that when flushOnTXEnd is *on* but asyncWriting is *off*, output will only be written when the buffer is full.
+This may take several hours, or even require a rsyslog shutdown. However, a buffer flush can be forced in that case by sending rsyslogd a HUP signal.
+
+=============  ===================
+flushOnTXEnd:  "on|off"
+Default:       "on"
+Legacy:        $OMFileFlushOnTXEnd
+=============  ===================
+
+Omfile has the capability to write output using a buffered writer.
+Disk writes are only done when the buffer is full. So if an error
+happens during that write, data is potentially lost.
+
+In cases where this is unacceptable, set this parameter to *on*.
+Then, data is written at the end of each transaction (for pre-v5 this means after each log
+message) and the usual error recovery thus can handle write errors
+without data loss. 
+Note that this option severely reduces the effect
+of zip compression and should be switched to *off* for that use case.
+
+=============  ===================
+ioBufferSize:  "size"
+Default:       "4K" (KB)
+Legacy:        $OMFileIOBufferSize
+=============  ===================
+
+Size (in KB) of the buffer used to write output data. The larger the
+buffer, the potentially better performance is. The default of 4K is
+quite conservative, it is useful to go up to 64k, and 128K if you
+used gzip compression (then, even higher sizes may make sense).
+   
+=========  ================
+dirOwner:  "username"
+Default:   *system default*
+Legacy:    $DirOwner
+=========  ================
+
+Set the owner for **newly created directories**. 
+This setting does not affect the owner of already existing directories.
+
+The parameter is a user name, for which the userid is
+obtained by rsyslogd during startup processing. Interim changes to
+the user mapping are not detected.
+
+================  ================
+dirOwnerNum:      "uid (number)"
+Default:          *system default*
+Available since:  7.5.8, 8.1.4
+================  ================
+
+**TODO why available since 2 versions? backport? bugfixes?**
```

Comment:
It will be good, for other doc pages, to know what's the current release/supported process.